<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>506</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_3">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>5</number>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLCDNumber" name="lcdNumber"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="3" column="0">
               <widget class="QPushButton" name="startbon">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                  <pointsize>17</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>按压</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="changebutton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="text">
                 <string>给予通关区间</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QWidget" name="level" native="true">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="2" column="1">
                  <widget class="QRadioButton" name="hard">
                   <property name="font">
                    <font>
                     <family>Broadway</family>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>hard</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QRadioButton" name="insane">
                   <property name="font">
                    <font>
                     <family>Broadway</family>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Insane</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QRadioButton" name="demon">
                   <property name="font">
                    <font>
                     <family>Broadway</family>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Demon</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QRadioButton" name="eazy">
                   <property name="font">
                    <font>
                     <family>Broadway</family>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>eazy</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0" colspan="2">
                  <widget class="QLabel" name="levellabel">
                   <property name="font">
                    <font>
                     <family>Curlz MT</family>
                     <pointsize>13</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>请选择反应难度^_^</string>
                   </property>
                   <property name="margin">
                    <number>7</number>
                   </property>
                   <property name="indent">
                    <number>20</number>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0" colspan="2">
                  <widget class="QLabel" name="rebonlabel">
                   <property name="font">
                    <font>
                     <pointsize>7</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>重置后需要再次点击！</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QPushButton" name="button">
                <property name="text">
                 <string>认输</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QPushButton" name="readyBon">
                <property name="text">
                 <string>准备</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>150</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <widget class="QLabel" name="charabacklabel">
              <property name="geometry">
               <rect>
                <x>9</x>
                <y>9</y>
                <width>16</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="obj1label">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>120</y>
                <width>91</width>
                <height>81</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="pixmap">
               <pixmap resource="image.qrc">:/Photo/obj1.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>90</y>
                <width>71</width>
                <height>71</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>10</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>200</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="image.qrc">:/Photo/character.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QProgressBar" name="proBar">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>381</width>
                <height>23</height>
               </rect>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
              <property name="textDirection">
               <enum>QProgressBar::TopToBottom</enum>
              </property>
             </widget>
             <widget class="QLabel" name="obj2label">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>130</y>
                <width>71</width>
                <height>61</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="image.qrc">:/Photo/obj1.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
             <zorder>obj2label</zorder>
             <zorder>obj1label</zorder>
             <zorder>charabacklabel</zorder>
             <zorder>proBar</zorder>
             <zorder>label</zorder>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="overlabel">
             <property name="text">
              <string>游戏失败</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rebon">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QPushButton" name="OKButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>幼圆</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>以下一难度返回！</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="OKButton2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>幼圆</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>不！保留原难度</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="OKlabel">
             <property name="font">
              <font>
               <family>方正姚体</family>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="text">
              <string>此模式已通关！</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QFormLayout" name="formLayout">
       <item row="2" column="1">
        <widget class="QPushButton" name="Mainbutton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>反应模式</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="startlabel">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="image.qrc">:/Photo/main.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="NorBon">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>正常模式</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
